### ISSUE TITLE
Different bidding on the same board

### ISSUE THREAD

**Issue by ThorvaldAagaard** [lorserker/ben#40]
I was going to test #38  on the training data provided by you in this Github repository.

So in the gameserver i just changed 
```
    rdeal = game.random_deal()
```
to 
```
    # rdeal format: ('96.9843.972.JT76 852.75.KT63.AK42 J3.KQJ62.AQJ.Q93 AKQT74.AT.854.85', 'W E-W')
    rdeal = ('962.73.AKQ83.T83 AJ7.AK982.T6.Q73 Q85.QJ54.J9754.K KT43.T6.2.AJ9642', 'S None')
```
And started the gameserver again.

Now I got Souths hand, but I was surprised, when I saw the bidding, as that was different

![image](https://github.com/lorserker/ben/assets/10560652/9d0dfc09-23bf-45ab-be53-f7eb6ede421e)

Later in the play I got some DDS-error, so I verified it was working with random deals, and once more tried feeding the gameserver with this specific deal, I now I got this bidding

![image](https://github.com/lorserker/ben/assets/10560652/a9bec424-e6e1-4f74-bfe8-0108963f2e03)

Looking in the log file I could see, that both Pass and 2C was considered
![image](https://github.com/lorserker/ben/assets/10560652/79603c80-dcdc-4d08-8959-8e8ae592fb16)

I can see that in this code
![image](https://github.com/lorserker/ben/assets/10560652/3a92e5c7-df1d-4df8-bb7f-e231c308ad8c)
the bid is selected with the highest expected_score and not from the neural network

What really confused me was that running the same deal once again the bot bid to 4S, using 2C

![image](https://github.com/lorserker/ben/assets/10560652/21a21738-b109-47b2-93a9-f4261b7c26f6)

West also considered 3N as the final contract with this result

![image](https://github.com/lorserker/ben/assets/10560652/4b0f3354-2fdc-47fc-a14d-ad1e1c585ca6)

Is this due to creating different sample hands?

P.W: Against 1N BEN continued with KA of diamonds before switching to a heart. But that is understandable.

**Comment by lorserker** [comment]
thanks @ThorvaldAagaard

**Comment by ThorvaldAagaard** [comment]
In #41 I have suggested how to seed the random function, thus solving this issue, so I am going to close this
